charmhelpers/contrib/openstack/context.py

```diff
@@ -768,6 +768,8 @@ def __call__(self):
         if send_notifications_to_logs:
             ctxt['send_notifications_to_logs'] = send_notifications_to_logs
 
+        ctxt['rabbit_heartbeat_timeout_threshold'] = conf.get('rabbit_heartbeat_timeout_threshold')
```

Comment:
in the charm's configuration we should use `rabbit-heartbeat-timeout-threshold` to keep the consistency.

Reply:
done